src/main/java/org/akaza/openclinica/service/ImportServiceImpl.java

```diff
@@ -1409,6 +1409,10 @@ private Object validateItem(ImportItemDataBean itemDataBean, CrfBean crf, EventC
                 return new DataImportReport(null, null, null, null, null, null, null, null, UPDATED, sdf_logFile.format(new Date()), null);
             }
         } else {
+        	// only created new event crf once
+        	if(eventCrf.getEventCrfId() == 0) {
+        		eventCrf = eventCrfDao.saveOrUpdate(eventCrf);	
+        	}        	
```

Comment:
move this logic to createItemData method so it will be applied to all places that call this method

Reply:
done